### PR TITLE
cpu/stm32l0 : timeout added on waiting loop

### PR DESCRIPTION
**Pull request by Marc-Aurele**
A timeout has been added to get out of busy waiting loop. I have already face issues where my cpu was blocked in this kind of loop.

Signed-off-by: Aurelien Fillau <aurelien.fillau@gmail.com>

### CODE REVIEW

**Comment by aabadie** [comment]
You have a cppcheck issue raised by travis, can you have a look ?

**Comment by Marc-Aurele** [comment]
Done

File: cpu/stm32l0/periph/i2c.c [hunk 1]
```diff
@@ -403,7 +403,7 @@ int i2c_write_regs(i2c_t dev, uint8_t address, uint8_t reg, const void *data, in
     }
 
     /* Check to see if the bus is busy */
-    while ((i2c->ISR & I2C_ISR_BUSY) & tick--) {
```
Comment (kaspar030) [inline cpu/stm32l0/periph/i2c.c]: this looks it was a pretty mean bug before.

Comment (Marc-Aurele) [reply cpu/stm32l0/periph/i2c.c]: yes, i don't know why i didn't noticed it before since i'm using I2C in a significant way with a lot of devices :s

**Pull request merged**